tests/test_userhistory.py

```diff
@@ -1,33 +1,44 @@
-""" Tests for user history class """
+''' Tests for user history class '''
```

Comment:
Consider using double quotes to keep consistent with the rest of the codebase -
I'm having some trouble here recalling whether we decided to use double or single quotes for docstrings. If we could all discuss this again briefly over discord or in a github conversation I would appreciate having it in writing to look back at in moments like these, sorry for being uncertain of where we landed in our verbal discussion.